code/index.html

```diff
@@ -2,17 +2,112 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project Name</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width-device-width, initial-scale=1.0">
+    <title>Lolas little treasure box</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="icon" href="Photos/favicon.ico" type="image/icon">
+   
   </head>
   <body>
     <header>
-      <!-- If you want a header with a title, logo and a navbar, here is where you should add that. -->
+      <div class="hamburger">
+          <div></div>
+          <div></div>
+          <div></div>
+      </div>
+      <img src="Photos/Logo.png" alt="Lola">
+      <nav>
+          <a href="#c1"> <strong>Das bin ich, <mark>Lola</mark></strong> </a>
+          <a href="#c2"> Meine kleinen Freuden</a>
+          <a href="#c3">Meine Lieblingswesen</a>
+      </nav>
     </header>
 
-    <main class="">
+<!--Coverbox-->
+<div class="cover">
+  <div class="container">
+  <section class="row">
+    <div class="column">
+      <h1 style="font-size: 300%;">Hi, I'm Lola.</h1>
+      <p style="font-size: 150%; font-weight: 100;">i'm a tiny human on this planet, trying to make sense of this so-called world. <br> 
+        one thought at the time. <br>
+        <br>
+        my biggest achievement up until now: loving the thought of birthdays.
+        <br>
+        i can walk.
+      </p>
+    </div>
+    <div class="column" style="display: flex; flex-direction: row-reverse; " >
+        <img src="Photos/pp.jpg" alt="Lola" style="width: 60%;" >
+    </div>
+</div>
+</section>
+</div>
+</div>
+
+
+    <main class="master">
       <!-- The main container to use as the flex parent for your card layout, the main content of the site should be in here.  -->
+      <div class="container1" id="c1">
+        <h2> Das bin ich </h2>
+          <section class="row">
+            <div class="column" style="margin-right: 4px;">
+              <img src="Photos/a_banana.jpg" alt=""  style="width: 100%;">
+              <img src="Photos/a_walls.jpg" alt=""  style="width: 100%;">
+            </div>
+            <div class="column"style="margin-left: 4px;">
+              <img src="Photos/h_sofadad.jpg" alt=""  style="width: 100%;">
+              <p style="font-weight: 400; text-align: center; width: 100%;"> we are style</p>
+            </div>
+          </section>
+      </div>
```

Comment:
Almost correctly indented ;) You should always include the alt attributes, but if you don't plan on writing anything it's better to omit them (and again - styling in the CSS file).

```
      <div class="container1" id="c1">
        <h2> Das bin ich </h2>
        <section class="row">
          <div class="column">
            <img src="Photos/a_banana.jpg">
            <img src="Photos/a_walls.jpg">
          </div>
          <div class="column">
            <img src="Photos/h_sofadad.jpg">
            <p> we are style</p>
          </div>
        </section>
      </div>
```

Reply:
Got it.